.github/workflows/build_and_test.yml

```diff
@@ -42,10 +42,10 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v2
         with:
-          node-version: "15"
-      - uses: pnpm/action-setup@v1.2.1
+          node-version: "16"
+      - uses: pnpm/action-setup@v2.0.1
         with:
-          version: 5.14.1
+          version: 6.15.0
       - run: pnpm add -g @vue/cli
       - run: pnpm install --frozen-lockfile
       - run: pnpm run test:vue3
```

Comment:
can't we just have one test stage that runs `pnpm run test` rather than a test_vue2 and test_vue3 stage?

Reply:
Sure we can do it this way.
I thought that two sets might be more helpful to find the root of an issue.

But yea it will simplify things only having a single step.

Reply:
Part of 6dee007